Algorithm-Pig-Latin.md

```diff
@@ -117,40 +157,33 @@ translatePigLatin("consonant");
 
 :rocket: [Run Code](https://repl.it/CLmv/0)
 
-### Fourth Solution (recursive and declarative)
```

Comment:
I would prefer this answer over any of the others, the other 3 uses regex, this is the only one that does not.